.internal/templates/services/wireguard/template.yml

```diff
@@ -1,23 +1,24 @@
 wireguard:
-  image: linuxserver/wireguard
   container_name: wireguard
-  cap_add:
-    - NET_ADMIN
-    - SYS_MODULE
+  image: ghcr.io/linuxserver/wireguard
+  restart: unless-stopped
   environment:
-    - PUID=1000
-    - PGID=1000
-    - TZ=Etc/UTC
-    - SERVERURL=Unset
-    - SERVERPORT=Unset
-    - PEERS=1
-    - PEERDNS=auto
-    - INTERNAL_SUBNET=100.64.0.0/24
-  volumes:
-    - ./services/wireguard/config:/config
-    - /lib/modules:/lib/modules
+  - PUID=1000
+  - PGID=1000
+  - TZ=Etc/UTC
+  - SERVERURL=Unset
+  - SERVERPORT=Unset
+  - PEERS=laptop,phone,tablet
+  - PEERDNS=auto
+# - INTERNAL_SUBNET=100.64.0.0/24
+  - ALLOWEDIPS=0.0.0.0/0
   ports:
-    - "51820:51820/udp"
+  - "51820:51820/udp"
+  volumes:
+  - ./volumes/wireguard:/config
+  - /lib/modules:/lib/modules:ro
+  cap_add:
+  - NET_ADMIN
+  - SYS_MODULE
   sysctls:
-    - net.ipv4.conf.all.src_valid_mark=1
-  restart: unless-stopped
+  - net.ipv4.conf.all.src_valid_mark=1
```

Comment:
Is this the correct indentation? YAML is sometimes confusing.

Reply:
In context, it seems OK to me. Here's the full thing:

```
wireguard:
  container_name: wireguard
  image: ghcr.io/linuxserver/wireguard
  restart: unless-stopped
  environment:
  - PUID=1000
  - PGID=1000
  - TZ=Etc/UTC
  - SERVERURL=Unset
  - SERVERPORT=Unset
  - PEERS=laptop,phone,tablet
  - PEERDNS=auto
# - INTERNAL_SUBNET=100.64.0.0/24
  - ALLOWEDIPS=0.0.0.0/0
  ports:
  - "51820:51820/udp"
  volumes:
  - ./volumes/wireguard:/config
  - /lib/modules:/lib/modules:ro
  cap_add:
  - NET_ADMIN
  - SYS_MODULE
  sysctls:
  - net.ipv4.conf.all.src_valid_mark=1
```

Yes/no?